validation/src/test/scala/hmda/validation/rules/lar/quality/Q039Spec.scala

```diff
@@ -0,0 +1,35 @@
+package hmda.validation.rules.lar.quality
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.rules.EditCheck
+import hmda.validation.rules.lar.{ BadValueUtils, LarEditCheckSpec }
+import org.scalacheck.Gen
+
+class Q039Spec extends LarEditCheckSpec with BadValueUtils {
+
+  property("All lars with HOEPA status and action taken type not equal to 1 must pass") {
```

Comment:
It might be worthwhile to break this into 2 tests, one for HOEPA and one for action taken type.